trio/socket.py

```diff
@@ -705,3 +741,5 @@ def _make_simple_sock_method_wrapper(methname, wait_fn, maybe_avail=False):
 #     else:
 #         raise OSError("getaddrinfo returned an empty list")
 # __all__.append("create_connection")
+
+
```

Comment:
Read until here so far. I'm not going to assume I can wrap all that in my head, but I did not saw anything particularly wrong. For now I'm going to switch task a bit.